### PR TITLE
7pm b slack bot update admin page

### PR DESCRIPTION
**Pull request by XuperBryan**
Updated frontend so that there is an additional tab in "admins" which will be used to manage teams. Currently uses placeholder text while waiting for CSV file upload operations to be finished.

### CODE REVIEW

**Review by gracsoo — APPROVED** [review]
everything looks good to me :)

**Comment by XuperBryan** [comment]
I estimate this as a 1 point story

**Review by gracsoo — APPROVED** [review]
everything looks good to me :D

File: javascript/.env [hunk 1]
```diff
@@ -1 +1,2 @@
-NODE_PATH=src
\ No newline at end of file
+NODE_PATH=src
+MONGODB_URI=mongodb+srv://readOnlyB:NGyb0MCUdaT8I8d4@cluster0.6bwgb.mongodb.net/slack?retryWrites=true&w=majority
```
Comment (btk5h) [inline javascript/.env:2]: I believe this is secret information, should this really be checked into version control?

**Review by pconrad — APPROVED** [review]
LGTM.

**Comment by pconrad** [comment]
This has two separate 20 points stories in it: so total 40 points.

**Pull request merged**